library/core/src/intrinsics.rs

```diff
@@ -2550,3 +2550,13 @@ pub const unsafe fn write_bytes<T>(dst: *mut T, val: u8, count: usize) {
         write_bytes(dst, val, count)
     }
 }
+
+#[cfg(not(bootstrap))]
+extern "rust-intrinsic" {
```

Comment:
Can you add the intrinsics to the main `extern "rust-intrinsic"` block instead and just cfg the fn directly?